### PR TITLE
raster scaling: allow default offset / scale value

### PR DESCRIPTION
**Pull request by jdroenner**

### CODE REVIEW

**Comment by jdroenner** [comment]
[Read GDAL dedicated scale and offset values](https://trello.com/c/d4iqIfAX/1220-read-gdal-dedicated-scale-and-offset-values)

File: projects/core/src/lib/backend/operator.model.ts [hunk 1]
```diff
@@ -315,8 +315,8 @@ export type InputResolutionDict = {type: 'source'} | {type: 'value'; x: number;
 export interface RasterUnScalingDict extends OperatorDict {
     type: 'RasterScaling';
     params: {
-        slope: RasterMetadataKey | {type: 'constant'; value: number};
-        offset: RasterMetadataKey | {type: 'constant'; value: number};
+        slope: RasterMetadataKey | {type: 'constant'; value: number} | undefined;
```
Comment (michaelmattig) [inline projects/core/src/lib/backend/operator.model.ts:318]: maybe use the ?-notation instead? `slope?: ...`

Comment (jdroenner) [reply projects/core/src/lib/backend/operator.model.ts]: [0305d3c](https://github.com/geo-engine/geoengine-ui/pull/584/commits/0305d3ccc12fb907ee7fdcfc3d04307180cc2768)

**Review by michaelmattig — APPROVED** [review]

**Comment by michaelmattig** [comment]
bors merge

**Pull request merged**